examples/config.yaml

```diff
@@ -27,3 +27,9 @@ pod-spec-patch:
         secretKeyRef:
           name: github-secrets
           key: github-token
+    resources:
+      requests:
+        cpu: 100m
+        mem: 50Mi
+      limits:
+        mem: 1Gi
```

Comment:
Did you want a test that includes something comma separated just to cover that case? Or does it not really matter that much?